wpilib/trajectory/__init__.py

```diff
@@ -1,7 +1,5 @@
-from . import _init_trajectory
-
-# autogenerated by 'robotpy-build create-imports wpilib.trajectory wpilib.trajectory._trajectory'
-from ._trajectory import (
+# autogenerated by 'robotpy-build create-imports wpilib.controller wpilib.controller._controller.trajectory'
```

Comment:
Hm, can we make `robotpy-build create-imports wpilib.trajectory wpilib.controller._controller.trajectory` work?